stdlib/REPL/src/docview.jl

```diff
@@ -299,8 +301,33 @@ function summarize(io::IO, T::DataType, binding::Binding)
     end
 end
 
-function summarize(io::IO, m::Module, binding::Binding)
-    println(io, "No docstring found for module `", m, "`.\n")
+function find_readme(m::Module)::Union{String, Nothing}
+    isnothing(pathof(m)) && return nothing
+    path = dirname(pathof(m))
+    top_path = pkgdir(m)
+    while true
+        for readme in ["README.md", "readme.md", "Readme.md", "ReadMe.md"]
+            readme_path = joinpath(path, readme)
+            if isfile(readme_path)
+                return readme_path
+            end
+        end
+        path == top_path && break # go no further than pkgdir
+        path = dirname(path) # work up through nested modules
+    end
+    return nothing
+end
+function summarize(io::IO, m::Module, binding::Binding; nlines::Int = 200)
+    readme_path = find_readme(m)
+    if !isnothing(readme_path)
+        println(io, "No docstring found for module `", m, "`.\n")
```

Comment:
This white space especially is i think excessive.
Because the next sentence: that says "Dispaying readme..." is a direct continuation from this.
```suggestion
        println(io, "No docstring found for module `$m`.")
```

Also is there a reason for not using interpolation?

Reply:
just from copying from original code. Updated